blog/draft/zo-min-mogelijk-npm-dependencies.md

```diff
@@ -0,0 +1,235 @@
+---
+authors: [tom-ootes]
+tags: [npm, javascript, typescript, nodejs, pnpm, dependabot, htmx]
+draft: true
+---
+# Voorkom JavaScript-moeheid: minimaliseer het aantal NPM packages
+
+Waarom je zou moeten streven naar zo min mogelijk npm packages, en hoe je dat doet.
```

Comment:
Kun je hier iets meer zinnen van maken? Bijvoorbeeld "in deze blogpost vertel ik blabla". Want 1 zin op het overzicht ziet er een beetje karig uit.